Makefile

```diff
@@ -0,0 +1,14 @@
+# Set these to the desired values
+ARTIFACT_ID=ldap
+VERSION=1.3.0
```

Comment:
I take this is a copy-pasta mistake? Shouldn't the `VERSION` correspond to the dogu version (in this case `2.4.47-1`)

Reply:
Yes, you are right. Changed this.